core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -308,18 +312,28 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     gcsDAO.deleteProject(projectName).map(_ => RequestComplete(StatusCodes.OK))
   }
 
-  def unregisterBillingProject(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
-    val isDeleted = dataSource.inTransaction { dataAccess =>
-      withBillingProject(projectName, dataAccess) { project =>
-        dataAccess.rawlsBillingProjectQuery.delete(project.projectName)
-      }
+  def registerBillingProject(projectName: RawlsBillingProjectName,
+                             owner: RawlsUserEmail,
+                             bucket: String): Future[PerRequestMessage] = {
+    val project = RawlsBillingProject(projectName, bucket, CreationStatuses.Ready, None, None)
+
+    for {
+      _ <- samDAO.createResource(SamResourceTypeNames.billingProject, projectName.value, userInfo)
+      _ <- samDAO.overwritePolicy(SamResourceTypeNames.billingProject, projectName.value, workspaceCreatorPolicyName, SamPolicy(Seq.empty, Seq.empty, Seq(SamProjectRoles.workspaceCreator)), userInfo)
+      _ <- samDAO.overwritePolicy(SamResourceTypeNames.billingProject, projectName.value, canComputeUserPolicyName, SamPolicy(Seq.empty, Seq.empty, Seq(SamProjectRoles.batchComputeUser, SamProjectRoles.notebookUser)), userInfo)
+      _ <- dataSource.inTransaction { dataAccess => dataAccess.rawlsBillingProjectQuery.create(project) }
+    } yield {
+      RequestComplete(StatusCodes.Created)
     }
+  }
 
-    //make sure we actually deleted it in the rawls database before destroying the permissions in Sam
-    isDeleted.flatMap {
-      case true => samDAO.deleteResource(SamResourceTypeNames.billingProject, projectName.value, userInfo)
-      case false => throw new RawlsExceptionWithErrorReport(errorReport = ErrorReport(StatusCodes.InternalServerError, s"Could not delete billing project [${projectName.value}]"))
-    }.map(_ => RequestComplete(StatusCodes.OK))
+  // NB: Not deleting project from Sam as that operation is not supported
+  def unregisterBillingProject(projectName: RawlsBillingProjectName): Future[PerRequestMessage] = {
+    for {
+      _ <- dataSource.inTransaction { dataAccess => dataAccess.rawlsBillingProjectQuery.delete(projectName) }
+    } yield {
+      RequestComplete(StatusCodes.NoContent)
+    }
   }
```

Comment:
@dvoet Is there anything else to handle above (now that we are not attempting to delete the project on the Sam side)?

Reply:
I would not even implement the unregisterBillingProject end point. I am afraid we will forget it is an incomplete operation.

Reply:
Do you recommend removing both the end point and the method above or?

Reply:
remove both